pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -208,8 +236,10 @@ func TestReconcile(t *testing.T) {
 	defer cancel()
 	c := testAssets.Controller
 	clients := testAssets.Clients
+	reconciler := c.Reconciler.(*Reconciler)
+	fr := reconciler.Recorder.(*record.FakeRecorder)
```

Comment:
This should be updated the same way it was done for the taskrun reconciler tests